templates/monasca-notification.service.j2

```diff
@@ -1,13 +1,15 @@
 [Unit]
 Description=Monasca Notification
-After=kafka.service postfix.service
```

Comment:
I would like to keep kafka and postfix specified. In the case where these are on the same box this is useful, in the case where they aren't systemd properly handles it and skips them.